src/test/java/seedu/address/testutil/AppointmentBuilder.java

```diff
@@ -0,0 +1,104 @@
+package seedu.address.testutil;
+
+import static seedu.address.testutil.TypicalPatients.ALICE;
+
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.Set;
+
+import seedu.address.model.appointment.Appointment;
+import seedu.address.model.appointment.AppointmentStatus;
+import seedu.address.model.appointment.Description;
+import seedu.address.model.patient.Patient;
+import seedu.address.model.tag.Tag;
+import seedu.address.model.util.SampleDataUtil;
+
+/**
+ * A utility class to help with building Appointment objects.
+ */
+public class AppointmentBuilder {
+
+    public static final Patient DEFAULT_PATIENT = ALICE;
+    public static final String DEFAULT_DATETIME = "2020-12-12T23:59:59";
```

Comment:
AppointmentBuilder fields need to change to match new implementation of Appointment